.pylintrc

```diff
@@ -7,6 +7,7 @@
 # R0913: too-many-arguments
 # R0902: too-many-instance-attributes
 # R0801: duplicate-code
-disable=C0111, R0903, W1203, E0611, E0401, R0913, R0902
+# R0201: no-self-use
```

Comment:
We can use [@staticmethod](http://missions5.blogspot.com/2014/12/python-classmethod-and-staticmethod.html)
to handle R201

Reply:
OK, fix in b36cae8.